pycon-jamaica-2016/videos/a-non-coders-journey-with-python-and-postgresql-rhys-stewart.json

```diff
@@ -0,0 +1,25 @@
+{
+  "copyright_text": "Standard YouTube License",
+  "description": "Python is sometimes thought of as the language for the casual coder. Rhys Stewart, self declared \"non-coder\" and SQL junkie (which means he can do crazy things with databases) will share his experiences of using Python with Postgresql for day to day tasks. He's not promising best practices but he says they are \"pretty good\" and work for him.\n\nhttps://www.pythonjam.org.jm/conference-2016 the PyCon Jamaica 2016 technology conference for Python software developers was held (November 17 - 18, 2016).  The conference was immediately followed by the Great Python Scavenger Hunt https://www.pythonjam.org.jm/conference-2016/scavenger-race. Our conferences are open to beginners, experts and non-programmers.",
+  "duration": 2851,
+  "language": "eng",
+  "recorded": "2016-11-25",
```

Comment:
Looks like this was on 2016-11-18 (https://www.pythonjam.org.jm/conference-2016/schedule)